packages/react/src/components/SignOutButton.tsx

```diff
@@ -1,25 +1,29 @@
-import type { SignOutCallback, SignOutOptions } from '@clerk/types';
+import type { SignOutOptions } from '@clerk/types';
 import React from 'react';
 
 import type { WithClerkProp } from '../types';
 import { assertSingleChild, normalizeWithDefaultValue, safeExecute } from '../utils';
 import { withClerk } from './withClerk';
 
 export type SignOutButtonProps = {
-  signOutCallback?: SignOutCallback;
+  redirectUrl?: string;
   signOutOptions?: SignOutOptions;
   children?: React.ReactNode;
 };
 
 export const SignOutButton = withClerk(
   ({ clerk, children, ...props }: React.PropsWithChildren<WithClerkProp<SignOutButtonProps>>) => {
-    const { signOutCallback, signOutOptions, ...rest } = props;
+    const { redirectUrl = '/', signOutOptions, ...rest } = props;
```

Comment:
Open to suggestions to moving this default to somewhere else (if necessary)

Reply:
🤔 I would suggest we open a ticket and move all the defaults (we have defaults for `afterSignInUrl`, `afterSingUpUrl`) to a separate module to have all the default settings in a centralized place. For it seems fine to leave it there.

Reply:
Done in SDK-1079